Packs/RoksitDNSSecurityIntegration-Sarp/Integrations/RoksitDNSSecurityIntegrationSarp/RoksitDNSSecurityIntegrationSarp.yml

```diff
@@ -0,0 +1,39 @@
+category: Utilities
+commonfields:
```

Comment:
The name needs to change, you can not have your name in it, either the word "integrations".
after changing the name, please go over the entire pack, and change all files name, ans content.